src/osd/OSD.cc

```diff
@@ -5942,7 +5943,8 @@ OSDService::ScrubJob::ScrubJob(const spg_t& pg, const utime_t& timestamp, bool m
       sched_time += rand() % (int)(g_conf->osd_scrub_min_interval *
```

Comment:
By not also having a per-pool osd_scrub_min_interval, setting a higher pool_scrub_interval will confuse the user if a scrub happens earlier than the pool setting.  Also, if the global osd_scrub_min_interval > pool_scrub_interval then the pool_scrub_interval will be ignored.

While osd_scrub_min_interval > osd_scrub_max_interval could be arguably an invalid configuration we don't warn about it, and it just schedules scrubs every osd_scrub_min_interval no matter the setting of the other.

I suggest we add both a pool_scrub_min_interval and a pool_scrub_max_interval instead of just a pool_scrub_interval.